slices/coreutils.yaml

```diff
@@ -48,6 +48,13 @@ slices:
       - coreutils_user-information
       - coreutils_working-context
 
+  # separate slice for env binary
+  env:
```

Comment:
don't forget the test.

Reply:
I didn't forget. [smoke.sh](https://github.com/canonical/chisel-releases/blob/ubuntu-24.10/tests/spread/integration/coreutils/smoke.sh) already tests all coreutils slices.

Reply:
ah that's a clever trick. Ok then lgtm ;)